### ISSUE TITLE
simple refactorings without wrangler

### ISSUE THREAD

**Issue by vladdu** [erlide/erlide#223]
can we implement some simple refactorings without wrangler?

- rename (local, at least)
- extract variable
- extract function

**Comment by martinpritchard** [comment]
Also, rename module - would be nice if it renamed the -module directive at the top of of the .erl file.  Rename variable would be extremely useful, especially for records (I'm thinking gen_server #state records in particular)

**Comment by vladdu** [comment]
Yes, thanks. The current model of the code is not fit for these refactorings, but I hope that I will be ready soon with an improved code base that allows that.
